modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/ProfileEpisodeListFragment.kt

```diff
@@ -451,6 +455,22 @@ class ProfileEpisodeListFragment : BaseFragment(), Toolbar.OnMenuItemClickListen
         }
     }
 
+    private fun onManageDownloadsMoreOptions() {
+        val dialog = OptionsDialog()
+        dialog.setTitle(resources.getString(LR.string.need_to_free_up_space))
+        dialog.addTextOption(LR.string.dismiss_manage_download_banner, click = this::onDismissManageDownloadTapped)
+        dialog.show(parentFragmentManager, "manage_downloads_more_options")
```

Comment:
```suggestion
        OptionsDialog()
            .setTitle(resources.getString(LR.string.need_to_free_up_space))
            .addTextOption(LR.string.dismiss_manage_download_banner, click = this::onDismissManageDownloadTapped)
            .show(parentFragmentManager, "manage_downloads_more_options")
```
This might look cleaner if the variable is not declared as it's not needed.